sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/datetimeExpressions.scala

```diff
@@ -595,7 +595,10 @@ case class WeekOfYear(child: Expression) extends UnaryExpression with ImplicitCa
 
 // scalastyle:off line.size.limit
 @ExpressionDescription(
-  usage = "_FUNC_(timestamp, fmt) - Converts `timestamp` to a value of string in the format specified by the date format `fmt`.",
+  usage = """
+    _FUNC_(timestamp, fmt) - Converts `timestamp` to a value of string in the format specified by the date format `fmt`.
+    See `java.text.SimpleDateFormat` for valid date and time format patterns.
```

Comment:
This isn't true. It's `DateTimeFormatter` in master.

Reply:
My bad here. Fixing this. Thanks for pointing this out